common/models/UserLocalSettings.ts

```diff
@@ -16,6 +17,12 @@ export type NotificationsSettings = {
     locals?: LocalNotificationsSchedule,
 };
 
+export type QolSettings = {
+    seenOnboardingQol?: boolean,
```

Comment:
I think I had mentioned this before, but there's not "onboarding qol".. that's just the first qol they take.

Reply:
So this should also be "seenQolOnboarding"

Reply:
@wynnset @MarkDavies0 is this resolved?